yt_dlp/extractor/youtube.py

```diff
@@ -66,6 +66,8 @@
     variadic,
 )
 
+
+STRAMING_DATA_CLIENT_NAME = '@@youtubeplzdontusethis@@client_name'
```

Comment:
```suggestion
STREAMING_DATA_CLIENT_NAME = '__yt_dlp_client'
```
is similar to what we use in jsinterp